Projects/Payment/Sources/Models/PaymentData.swift

```diff
@@ -78,6 +78,12 @@ public struct PaymentData: Codable, Equatable {
             self.bank = bank
         }
 
+        init(with model: OctopusGraphQL.PaymentInformationQuery.Data) {
```

Comment:
Can we move this init to octopus implementation?
Also if we can fill this data only if the member has connected payments method